examples/eip4844/signed_blinded_beacon_block.json

```diff
@@ -137,9 +137,24 @@
           "timestamp": "1",
           "extra_data": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2",
           "base_fee_per_gas": "1",
+          "excess_data_gas": "1",
           "block_hash": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2",
-          "transactions_root": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"
-        }
+          "transactions_root": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2",
+          "withdrawals_root": "0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"
+        },
+        "bls_to_execution_changes": [
```

Comment:
I think this is not in the blinded beacon block body?

Reply:
You are right, the beacon-API types don't include it for the `blinded_beacon_block`. Anyways make me wonder why is not the case?

Reply:
Nope I was wrong :)